### PR TITLE
A blog-post-like writeup of using one of our benchmarks for performance engineering

### LINKED ISSUE #8647: Create a "blog post" working through use of Benchmarkerator
As an exercise in both exploring the use of our benchmarking tools and documenting their use, attempt to use the tools to make some kind of performance improvement in one of our key performance metric operations (tentatively, creating a vault) while taking copious notes, screen snapshots, and data dumps throughout the whole exercise end to end, and then write it up the process and its findings in the form of something resembling a Medium post (though most likely simply to be published as a regular document on our development wiki).

This is in further service of #8327

### PR DESCRIPTION
**Pull request by FUDCo**
Note that there is no code in this PR.
However, it does depend on #8764 and #8765 landing in order to be truthful.

Nominally closes #8647

### CODE REVIEW

**Comment by FUDCo** [comment]
@warner I've tagged you as the reviewer, but feel free to add anyone you think might be relevant to look at this.

File: packages/benchmark/doc/usingBenchmarks/usingBenchmarks.md [hunk 1]
```diff
@@ -0,0 +1,974 @@
...
+
+Also, the command:
+
+`# tsx benchmark-vault-open.js --help`
+
+will give you a summary of all the various command line options.  Note that all
+benchmarks built with the Benchmarkerator tool have the same command line
+interface.)
+
+## Benchamark outputs
```
Comment (warner) [inline packages/benchmark/doc/usingBenchmarks/usingBenchmarks.md]: typo (or a nifty bit of new jargon)

**Review by warner — APPROVED** [review]
looks good, maybe add one note on the flamegraph section, a reminder that it has a non-zero sampling interval, and to not assume that a contiguous block of "stable" stack means that the function never returned during that interval, and that fast function calls won't appear at all

File: packages/benchmark/doc/usingBenchmarks/usingBenchmarks.md (hunk 1 above)
Comment (FUDCo) [reply packages/benchmark/doc/usingBenchmarks/usingBenchmarks.md]: Thatsa benchamark for ya!

**Pull request merged**